packages/connect-ui/src/views/Passphrase.tsx

```diff
@@ -100,7 +100,7 @@ export const Passphrase = (props: PassphraseProps) => {
                         title={
                             <FormattedMessage
                                 id="TR_WALLET_SELECTION_HIDDEN_WALLET"
-                                defaultMessage="Hidden wallet"
+                                defaultMessage="Passphrase wallet"
```

Comment:
I think important is to edit `messages.ts` and not in-code stuff,

Reply:
@jvaclavik this is `connect` (explorer?) Not sure how what Komret wrote applies here. So feel free to ignore my comment